bind-mount.c

```diff
@@ -241,7 +241,6 @@ parse_mountinfo (int  proc_fd,
   int max_id;
   unsigned int n_lines;
   int root;
-
```

Comment:
Please don't adjust whitespace in locations that you're not otherwise editing.

This blank line is intentional: bubblewrap is mostly written in a C89 style where variable declarations must come before statements, and when writing in that style it's common to have an empty line between the last variable declaration and the first statement.